### ISSUE TITLE
Re-construct information architecture

### ISSUE THREAD

**Issue by toni-sharpe** [toni-sharpe/brainstem-research#48]
### Problem

Some pages might not be logically organised

There is also a space issue in the menu

### Solution

A new page "Misc" should be added for things like the TimeLine and the Anti-Bias stuff can be factored out across relevant pages, aim, fewer top levels

The solution should also apply the rebuilt information architecture

- [ ] design (see /notes/ folder)
- [ ] re-arrange
- [ ] move sub-heading text into main body for PrimeSymptom page
- [ ] enable 2nd-tier navigation
- [ ] in-page filter prop, used to handle histogram settings, and TimeLine ones
- [ ] and component for the last `[x]`
